Dockerfile

```
@@ -63,8 +63,15 @@ RUN npm install yarn -g
 
 RUN echo "gem: --no-ri --no-rdoc --no-document" > /root/.gemrc
 
+RUN curl -o /usr/lib/rpm/brp-strip https://raw.githubusercontent.com/rpm-software-management/rpm/rpm-4.19.1-release/scripts/brp-strip
+
+RUN chmod +x /usr/lib/rpm/brp-strip
+
 COPY . /build_scripts
 
+RUN cd /usr/lib/rpm/ && \
+  patch -p2 < /build_scripts/container-assets/Add-js-rb-filtering-on-top-of-4.19.1.patch
+
```

You can still smash all these together further

```suggestion
COPY . /build_scripts

RUN curl -o /usr/lib/rpm/brp-strip https://raw.githubusercontent.com/rpm-software-management/rpm/rpm-4.19.1-release/scripts/brp-strip && \
  chmod +x /usr/lib/rpm/brp-strip && \
  cd /usr/lib/rpm/ && \
  patch -p2 < /build_scripts/container-assets/Add-js-rb-filtering-on-top-of-4.19.1.patch
```